commune_app/tests.py

```diff
@@ -1,3 +1,28 @@
-# from django.test import TestCase
+import pytest
 
-# Create your tests here.
+from . import models
+
+
+CHORE_TITLE = "Dishes"
+CHORE_DATE = (1997, 10, 19)
+CHORE_ASSIGN_TO = "Elias"
+CHORE_DESCRIPTION = "Clean the Dishes"
+
+
+@pytest.fixture
+def chore0():
+    return models.Chore(title=CHORE_TITLE, date=CHORE_DATE, assign_to=CHORE_ASSIGN_TO, description=CHORE_DESCRIPTION)
+
+
+class TestChoreModel:
+    def test_new_chore(self, chore0):
+        assert chore0.title == CHORE_TITLE
+        assert chore0.date == CHORE_DATE
+        assert chore0.assign_to == CHORE_ASSIGN_TO
+        assert chore0.description == CHORE_DESCRIPTION
+
+    @pytest.mark.django_db()
+    def test_persist_chore(self, chore0):
+        chore0.save()
+
+        assert chore0 in models.Chore.objects.all()
```

Comment:
I think you can add validation tests for each field like:
 - long title
 - not exist assign to
 - long description
 - empty fields